torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -21,21 +21,21 @@ static variable_list CopyBackwards_apply_functional(
     std::array<bool, 2> needs_input_grad,
     const c10::TensorOptions& src_options) {
   check_input_variables("CopyBackwards", grads, 1, -1, true);
-  auto grad = c10::MaybeOwned<at::Tensor>::borrowed(grads[0]);
```

Comment:
Not sure about these utilities TBH. What was the purpose of MaybeOwned and why is unconditionally moving better?